lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1970,5 +1970,11 @@ sub status_info {
     return $info;
 }
 
+sub test_name {
+    my ($self) = @_;
+
+    return $self->settings_hash->{JOB_TEMPLATE_NAME} || $self->TEST;
```

Comment:
I would remove this bad-performing function so nobody thinks it would be a good idea to call this in a loop. And besides, it is dead code now, right?

Reply:
yes, it is dead code, and I forgot to remove it.